ethers-providers/src/pubsub.rs

```diff
@@ -31,6 +31,8 @@ pub struct SubscriptionStream<'a, P: PubsubClient, R: DeserializeOwned> {
     /// The subscription's installed id on the ethereum node
     pub id: U256,
 
+    loaded_elements: Vec<R>,
```

Comment:
looks like we can use a VecDeque here to easily `pop_front`?

Reply:
good catch